WorkoutGeneratorApp/App.tsx

```diff
@@ -7,56 +7,35 @@
  *
  * @format
  */
-
-import React from 'react';
-import {
-  Text,
-  View,
-} from 'react-native';
-
-import { Provider as PaperProvider } from 'react-native-paper';
-
-import { NavigationContainer } from '@react-navigation/native';
-import { createNativeStackNavigator  } from '@react-navigation/native-stack';
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+// import { NavigationContainer } from '@react-navigation/native';
+import * as React from 'react';
```

Comment:
Should only import what you need. In this case, you are using `React.useState()` so it should really only be `import React from 'react';` or `import React, { useState } from 'react'` and then you can directly call `useState(...)` on line 19 and 20.

Reply:
true, but is this an issue I need to fix or just a tip?

Reply:
I would rather just have another commit for this small change instead of including it in another request or forgetting it, what do you think?